navigation/AppNavigator.js

```diff
@@ -24,7 +26,12 @@ class AuthLoadingScreen extends React.Component {
 
     // This will switch to the App screen or Auth screen and this loading
     // screen will be unmounted and thrown away.
-    this.props.navigation.navigate(userToken ? 'App' : 'Auth');
+
+    // Correct version
```

Comment:
Nitpick: could you add a "Temp. Signout" button to _somewhere_ in the app, so that we can use that to navigate back and forth? (Also why I asked @12aschen if there's a plan to implement "log/signout")

Reply:
I just worry that we will forgot to add auth back before pilot 😅 @wangannie